test/linodes/backups.bats

```diff
@@ -0,0 +1,105 @@
+load '../test_helper/bats-support/load'
+load '../test_helper/bats-assert/load'
+load '../common'
+
+# ##################################################################
+# #  WARNING: THIS TEST WILL DELETE ALL OF YOUR LINODES            #
+# #  WARNING: USE A SEPARATE TEST ACCOUNT WHEN RUNNING THESE TESTS #
+# ##################################################################
+
+setup() {
+    export timestamp=$(date +%s)
+    clean_linodes="false"
+    linode_id=$(linode-cli linodes list --format id --text --no-header | head -n 1)
+}
+
+teardown() {
+    unset timestamp
+
+    if [ "$clean_linodes" = "true" ];
+    then
+        run removeLinodes
+    fi
+}
+
+@test "it should create a linode with backups disabled" {
+    run createLinode
+    local linode_id=$(linode-cli linodes list --format id --text --no-header | head -n 1)
+    run linode-cli linodes backups-cancel $linode_id --text
```

Comment:
This is a kinda lousy way to assert that backups are enabled.  The CLI supports showing this. .but the attribute name is lousy: `linode-cli linodes list --format 'id,enabled'` shows you all your Linodes and if their backups are enabled.

Reply:
awesome, i was thinking the same when i wrote it, but at the time didn't know about the above. updating.